.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,22 @@
-<!--
-****Please describe the pull request motivation and changes here
-along with non-obvious things.
--->
+## Overview
+
+<!-- Please describe your changes here 
+  and list any open questions you might have. -->
+
+---
+<!-- This is for Exonum Team members only. -->
+<!-- markdownlint-disable MD034 -->
+See: https://jira.bf.local/browse/ECR-XYZW
+<!-- markdownlint-enable MD034 -->
+
+### Definition of Done
+
+- [ ] There are no TODOs left in the merged code
+- [ ] Change is covered by automated tests
+- [ ] Benchmark results are attached (if applicable)
+- [ ] The [coding guidelines] are followed
+- [ ] Public API has proper documentation
+- [ ] Changelog is updated if needed (in case of notable or breaking changes)
+- [ ] The continuous integration build passes
```

Comment:
What do you think about clause like?
`Difficult code parts contain explanatory comments.`
It is obvious?

Reply:
@alekseysidorov I suppose it should be described in code conduct.

Reply:
Well, not CoC, but coding guidelines should outline our stance on commenting the code.